content/using_ec2_spot_instances_with_eks/eksctl/prerequisites.md

```diff
@@ -2,12 +2,13 @@
 title: "Prerequisites"
 date: 2018-08-07T13:31:55-07:00
 weight: 10
+description: "using latest cli"
 ---
 
 For this module, we need to download the [eksctl](https://eksctl.io/) binary:
 ```
-export EKSCTL_VERSION=0.18.0
-curl --silent --location "https://github.com/weaveworks/eksctl/releases/download/${EKSCTL_VERSION}/eksctl_Linux_amd64.tar.gz" | tar xz -C /tmp
+curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
```

Comment:
I really don't think we should be doing this ; The pinning things to the latest version.  In the past this has come to bite us; Some of the versions that were out from eksctl had new functionality that made things incompatible and not to work over a period of time (or bugs in some cases);

Last thing we want is for those running an event or going through the workshop finding out that although they are on the latest version, there is a bug or something that makes it incompatible. This has already happen to us in the past and made us move to use deterministic versions.

Reply:
Appreciate the effort however ! really thanks a lot for contributing !